WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -455,9 +455,9 @@ private extension StorePickerViewController {
         }
 
         // If a site address was passed in credentials, select it
-        if let siteAddress = ServiceLocator.stores.sessionManager.defaultCredentials?.siteAddress,
+        if case let .wpcom(username: _, authToken: _, siteAddress: siteAddress) = ServiceLocator.stores.sessionManager.defaultCredentials,
```

Comment:
Nit: this can be simplified:

```suggestion
        if case let .wpcom(_, _, siteAddress) = ServiceLocator.stores.sessionManager.defaultCredentials,
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/8446/commits/b231bcef7029ee63fab0c1e723c9e645ed2e16e1